docs/docs/cmd/entra/m365group/m365group-user-list.mdx

```diff
@@ -33,6 +33,25 @@ m365 entra m365group user list [options]
 
 <Global />
 
+## Permissions
+
+<Tabs>
+  <TabItem value="Delegated">
+
+  | Resource        | Permissions                 |
+  |-----------------|-----------------------------|
+  | Microsoft Graph | GroupMember.Read.All |
+
+  </TabItem>
+  <TabItem value="Application">
+
+  | Resource        | Permissions                 |
+  |-----------------|-----------------------------|
+  | Microsoft Graph | GroupMember.Read.All |
+
+  </TabItem>
+</Tabs>
```

Comment:
This API appears to be a bit more complex. When an app only has `GroupMember.Read.All` permissions, it won’t return detailed user information, user objects will be `null`. I’d recommend including `User.ReadBasic.All` as the minimum permission for application-level access. (ref: https://learn.microsoft.com/en-us/graph/permissions-overview?tabs=http#limited-information-returned-for-inaccessible-member-objects)

The same applies to delegated scenarios, such as with `m365group user add`: https://learn.microsoft.com/en-us/graph/api/group-list-members?view=graph-rest-1.0&tabs=http#permissions